code.js

```diff
@@ -1,3 +1,39 @@
+const { check } = require("jsverify")
+
+
 function permutationSort(a) {
-    return 0;
+    right = false
+    if(a.length <= 1){
+        return [0, a]
+    }
+    arr = option(a, [])
+    return [0, arr]
 }
+
+function order(abb){
+    for(i = 0; i < abb.length-1; i++){
+        if(abb[i] > abb[i+1]){
+            return false
+        }
+    }
+    return true
+}
+
+function option(arr3, guess){
+    arr3.forEach((num, index) => {
+        if(right == false){
```

Comment:
If you're using a variable, please pass it as an argument. Otherwise things might break in unexpected ways.